mods/ra/maps/exodus/rules.yaml

```diff
@@ -0,0 +1,180 @@
+World:
+	MissionData:
+		Briefing: The Allies are now being flanked from both sides. Evacuate before the remaining Allied forces in the area are wiped out. Einstein has recently developed a technology which allows us to obscure units from the enemy. Pull out at least one prototype mobile gap generator intact.
+	LuaScript:
+		Scripts: exodus.lua
+	ScriptLobbyDropdown@difficulty:
+		ID: difficulty
+		Label: Difficulty
+		Values:
+			easy: Easy
+			normal: Normal
+			hard: Hard
+		Default: normal
+
```

Comment:
I guess to prevent people from cheating on missions by being able to see the whole map. But of course they can now also just look at them in editor. I guess there's really no point to it then.

Could I ask you to add a commit that removes [these two lines](https://github.com/OpenRA/OpenRA/blob/bleed/mods/ra/maps/evacuation/rules.yaml#L6-L7) then, please?